### DISCUSSION TITLE
Column names with prefix / suffix validation?

### DISCUSSION THREAD

**Discussion by rider87** [JSQLParser/JSqlParser#1259] [General]
Hi all,

I would like to have a SELECT like:
`String expression = "Select x FROM x WHERE [test] = 'Testuser' ";`

At rendering time I will change the placeholder [test] to the real Table / Column.
Currently the validation is failing, because JSQLParser cannot use the [ and ] so it will be throw an exception.

Is there a way to validate the SQL with this kind of placeholders?

Thanks

**Comment by manticore-projects** [comment]
Greetings.

As per default `[` `]` indicate Arrays, not field quotes.
You will need to set `SquareBracketQuotation` when calling the parser in order to allow TSQL `[` `]` quotes.

Please close the case when solved.

**Comment by rider87** [comment]
Thank you
